src/index.css

```diff
@@ -1,3 +1,270 @@
 * {
   box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: #fafafa;
+}
+
+nav {
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px 30px;
+}
```

Comment:
- [ ] Please add `border-bottom` to your navbar because it's not matching with [Zaplin](https://zeplin.io/) design :+1: check below :point_down:  the comparison between your design and [Zaplin](https://zeplin.io/) design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-06-23 16-16-01](https://github.com/husush23/Book-Store/assets/45226848/c11ee783-2e24-4180-8b81-7a446afd9add)

**Zaplin Design**
![Screenshot from 2023-06-23 16-16-14](https://github.com/husush23/Book-Store/assets/45226848/8420193d-3eeb-49fc-9216-e8fd279ed8b4)

- [ ] Also, Please increase the `border-radius` of your user icon your user icon is not the same as [Zaplin](https://zeplin.io/)

**Your Design**
![Screenshot from 2023-06-23 16-15-41](https://github.com/husush23/Book-Store/assets/45226848/be0318c1-2055-404e-a46e-bb707525386f)

**Zaplin Design**
![Screenshot from 2023-06-23 16-15-31](https://github.com/husush23/Book-Store/assets/45226848/a0d722d1-7c25-4730-b7a4-aeea36d61289)